### PR TITLE
Fix for issue #244

### PR DESCRIPTION
**Pull request by andreaskring**
Fixes issue #244

### CODE REVIEW

**Comment by chalkos** [comment]
please move the test from `test/java/com/databasepreservation/modules/siard/out/metadata/TestTableIndexFileStrategy.java` to `test/java/com/databasepreservation/testing/unit/modules/siard/out/metadata/TestTableIndexFileStrategy.java` if those are unit tests (they seem to be).

File: dbptk-core/src/test/java/com/databasepreservation/modules/siard/out/metadata/TestTableIndexFileStrategy.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  }
+
+  @Test
+  public void shouldEscapeWhenParentesisInWord() {
+    TableIndexFileStrategy tableIndexFileStrategy = new TableIndexFileStrategy(null);
+    assertEquals("\"Normal(Word\"", tableIndexFileStrategy.escapeString("Normal(Word"));
+  }
+
+  @Test(enabled = false)
+  public void test() {
```
Comment (chalkos) [inline dbptk-core/src/test/java/com/databasepreservation/modules/siard/out/metadata/TestTableIndexFileStrategy.java]: what is the purpose of this test?

**Comment by andreaskring** [comment]
@chalkos Regarding your comment about moving the test... this is not possible due to the access modifier of the method that is being tested. It is only visible from within the same package.

Are you refering to the last test seen in the code snippet above? If so, that test was only there to assure that the test was actually picked up by testng. I can remove it if you prefer this?

**Comment by chalkos** [comment]
@andreaskring my comment to move the file was an attempt to rearrange and organise the tests, but I forgot that very important detail. You can leave them where they are. I'll arrange them later after looking a bit into this problem and possible solutions.

I would like to avoid having tests that add no real test value, ie. that do not test anything in the application. A debug test is one of those kind of tests I would like to avoid having in master branch.

**Comment by andreaskring** [comment]
@chalkos Ok - I will remove the debug test

**Comment by andreaskring** [comment]
@chalkos Ok - I will remove the debug test

**Comment by andreaskring** [comment]
@chalkos The test has been removed now, so we should be ready to merge

**Pull request merged**